testOutput.ps1

```diff
@@ -1,5 +1,5 @@
 $version = $args[0]
-$formats = @("Html", "Dhtml", "Word", "Excel", "JSON")
+$formats = @("Html", "Dhtml", "Word", "Excel", "JSON", "Cucumber")
```

Comment:
Ah, my mistake - the output formats are defined in `testOutput.proj`, not in `testRunnersMsBuild.cmd`. Could you make the change there please?